cypress/e2e/focus.cy.js

```diff
@@ -0,0 +1,57 @@
+const waitTime = 500;
+const viewports = { "desktop": 'macbook-13', "mobile": 'iphone-6+' };
+const domain = 'http://localhost:3000'
+const focusTests = {
```

Comment:
@oluoluoxenfree Can we move this to a file outside of the test itself, so that we could specify additional tests easier. Maybe break it down to one component/url per file